src/rust/plugin-execution-sidecar/generator-execution-sidecar.rs

```diff
@@ -12,6 +12,9 @@ async fn main() -> Result<(), Box<dyn std::error::Error>> {
     let _guard = setup_tracing(SERVICE_NAME)?;
     let plugin_executor_config = PluginExecutorConfig::parse();
 
+    // Give the plugin a little time to become available.
+    tokio::time::sleep(std::time::Duration::from_secs(5)).await;
```

Comment:
Is there any way we can make this deterministic?

Reply:
discussion here
https://grapl-internal.slack.com/archives/C03JNJ57A9G/p1659471556377899

basically plugins and plugin-execution-sidecar may have a dependency cycle of health checks. fun!

Reply:
so i phoned it in for now